src/lib/components/Ideogram.react.js

```diff
@@ -477,15 +472,23 @@ Ideogram.propTypes = {
     dataDir: PropTypes.string,
 
     /**
-     * This is a work in progess and will hopefully be fixed in future releases.
+     * Organism(s) to show chromosomes for. Supply organism's name as a string (e.g. "human") or
+     * organism's NCBI Taxonomy ID (taxid, e.g. 9606) to display chromosomes from a single organism,
+     * or an array of organisms' names or taxids to display chromosomes from multiple species.
+     */
+    organism: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+
+    /**
+     * Provide local JSON organism into this prop from a local user JSON file.
+     * DataDir must not be initialized.
      */
-    heatmaps: PropTypes.arrayOf(PropTypes.object),
+    localOrganism: PropTypes.object,
 
     /**
      * Used to compare two chromosomes with each other.
-     * The keys "chrOne" and "chrTwo" represent one chromosome each. Organism is the 
-     * taxID or name. Start is an array, containing start one and 
-     * start two in this order. Stop is array, containing stop one, and stop two, 
+     * The keys "chrOne" and "chrTwo" represent one chromosome each. Organism is the
+     * taxID or name. Start is an array, containing start one and
+     * start two in this order. Stop is array, containing stop one, and stop two,
```

Comment:
```suggestion
     * start two in this order. Stop is array, containing stop one and stop two,
```
It seems that all these words would want quotes or back ticks, since they refer to keys or variable names, just like "chrOne" and "chrTwo"...